stem/utils.py

```diff
@@ -11,6 +11,8 @@
     from stem.mesh import Element, Mesh
     from stem.geometry import Geometry
 
+NUMBER_TYPES = (int, float, np.int64, np.float64)
+
```

Comment:
"""
TypeAlias:
    - NUMBER_TYPES: Union[int, float, np.int64, np.float64]
"""

Reply:
Arent you missing: numpy.float96 numpy.float128

Reply:
are those relevant?